src/risedevtool/src/task/compactor_service.rs

```diff
@@ -96,6 +96,13 @@ impl Task for CompactorService {
             cmd.env("MALLOC_CONF", conf); // unprefixed for linux
         }
 
+        if crate::util::is_env_set("CONNECTOR_LIBS_PATH") {
+            cmd.env(
+                "CONNECTOR_LIBS_PATH",
+                env::var("CONNECTOR_LIBS_PATH").unwrap(),
+            );
+        }
```

Comment:
Don't quite get that. Is this a no-op?

Reply:
Nope. `cmd.env` will set the env for the tmux session while `env::var("CONNECTOR_LIBS_PATH")` will get the host process env.

Reply:
Oh, I missed that. However, I guess it may not a good practice to export the "path" env var in `user.env`. As you can see, there're only boolean variables and they control whether the `risedev` will prepare or build some components, but usually have nothing to do with `risedev-dev` (`./risedev d`).

Reply:
Agree. We can remove configuration `CONNECTOR_LIBS_PATH` from `risingwave/src/risedevtool/config/src/main.rs` and add it to `src/risedevtool/src/task/compute_node_service.rs` and `src/risedevtool/src/task/meta_node_service.rs`. BTW, this configuration is not needed for frontend and compactor.

Reply:
> Agree. We can remove configuration `CONNECTOR_LIBS_PATH` from `risingwave/src/risedevtool/config/src/main.rs` and add it to `src/risedevtool/src/task/compute_node_service.rs` and `src/risedevtool/src/task/meta_node_service.rs`. BTW, this configuration is not needed for frontend and compactor.

+1.

Reply:
Addressed in the new commit.